src/autopas/containers/linkedCells/traversals/C01CudaTraversal.h

```diff
@@ -145,7 +145,6 @@ inline void C01CudaTraversal<ParticleCell, PairwiseFunctor, dataLayout, useNewto
 
 template <class ParticleCell, class PairwiseFunctor, DataLayoutOption dataLayout, bool useNewton3>
 inline void C01CudaTraversal<ParticleCell, PairwiseFunctor, dataLayout, useNewton3>::traverseParticlePairs() {
-  auto &cells = *(this->_cells);
```

Comment:
should not be removed, still needed by cuda!

Reply:
so move it further down

Reply:
Oh yes, sorry! Thanks!